README.md

```diff
@@ -1,17 +1,8 @@
 # Deis Monitor v2
-
-[![Build Status](https://travis-ci.org/deis/monitor.svg?branch=master)](https://travis-ci.org/deis/monitor)
-[![Docker Repository on Quay](https://quay.io/repository/deisci/monitor/status "Docker Repository on Quay")](https://quay.io/repository/deisci/monitor)
```

Comment:
Can we link to each of the three containers then?
[<sup>grafana</sup> ![Grafana Docker Repository on Quay](https://quay.io/repository/deis/grafana/status "Grafana Docker Repository on Quay")](https://quay.io/repository/deis/grafana)&nbsp;&nbsp;[<sup>telegraf</sup> ![Telegraf Docker Repository on Quay](https://quay.io/repository/deis/telegraf/status "Telegraf Docker Repository on Quay")](https://quay.io/repository/deis/telegraf)&nbsp;&nbsp;[<sup>influxdb</sup> ![InfluxDB Docker Repository on Quay](https://quay.io/repository/deis/influxdb/status "InfluxDB Docker Repository on Quay")](https://quay.io/repository/deis/influxdb)